frontend/javascripts/oxalis/view/jobs/train_ai_model.tsx

```diff
@@ -425,19 +468,43 @@ function AnnotationsCsvInput({
           name: annotation.dataSetName,
         });
 
-        const volumeTracings: ServerVolumeTracing[] = await Promise.all(
+        const volumeServerTracings: ServerVolumeTracing[] = await Promise.all(
           annotation.annotationLayers
-            .filter((layer) => layer.typ === "Skeleton")
+            .filter((layer) => layer.typ === "Volume")
             .map(
               (layer) =>
                 getTracingForAnnotationType(annotation, layer) as Promise<ServerVolumeTracing>,
             ),
         );
+        // TODO: make bboxs a member again
+        const volumeTracings = volumeServerTracings.map((tracing) =>
+          serverVolumeToClientVolumeTracing(tracing),
+        );
+        let userBoundingBoxes = volumeTracings[0]?.userBoundingBoxes;
+        if (!userBoundingBoxes) {
+          const skeletonLayer = annotation.annotationLayers.find(
+            (layer) => layer.typ === "Skeleton",
+          );
+          if (skeletonLayer) {
+            const skeletonTracing = await getTracingForAnnotationType(annotation, skeletonLayer);
+            userBoundingBoxes = convertUserBoundingBoxesFromServerToFrontend(
+              skeletonTracing.userBoundingBoxes,
+            );
+          } else {
```

Comment:
Could you add a comment here that user bounding boxes are always saved in all annotation layers, but if there is no volume annotation layer, the skeleton layer is checked. At least that's how I interpret it, but I was a bit confused at first :)